tests/test_linkedlist.py

```diff
@@ -1,6 +1,6 @@
 import io, pytest, sys
 
```

Comment:
You are changing to many things at the same time.  But now that you have changed the directory structure, you need to add an `__init__.py` file in the test folder too.